Fail to parse TypeScript errors when using tsc without the silent option in yarn

If I run `tsc` using yarn without silent option, I will get the following output.

```
yarn run v1.22.10
$ /Users/odan/source/github.com/odan-sandbox/reviewdog-later-strict-example/node_modules/.bin/tsc --project tsconfig.strict.json --noEmit
src/app.ts(1,29): error TS7006: Parameter 'x' implicitly has an 'any' type.
src/app.ts(5,30): error TS7006: Parameter 'x' implicitly has an 'any' type.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Analyzing this with [typescript's Errorformat](https://github.com/reviewdog/errorformat/blob/a74eb2146369f1c1c51b9ad32dea1a716eeb8a34/fmts/typescript.go#L9) will fail to parse it.
ref: https://reviewdog.github.io/errorformat-playground/?efms=%25E%25f+%25%23%28%25l%2C%25c%29%3A+error+TS%25n%3A+%25m&text=yarn+run+v1.22.10%0A%24+%2FUsers%2Fodan%2Fsource%2Fgithub.com%2Fodan-sandbox%2Freviewdog-later-strict-example%2Fnode_modules%2F.bin%2Ftsc+--project+tsconfig.strict.json+--noEmit%0Asrc%2Fapp.ts%281%2C29%29%3A+error+TS7006%3A+Parameter+%27x%27+implicitly+has+an+%27any%27+type.%0Asrc%2Fapp.ts%285%2C30%29%3A+error+TS7006%3A+Parameter+%27x%27+implicitly+has+an+%27any%27+type.%0Ainfo+Visit+https%3A%2F%2Fyarnpkg.com%2Fen%2Fdocs%2Fcli%2Frun+for+documentation+about+this+command.%0A


If yarn's silent option is enabled, only error messages will be printed and the parsing will succeed.
```
src/app.ts(1,29): error TS7006: Parameter 'x' implicitly has an 'any' type.
src/app.ts(5,30): error TS7006: Parameter 'x' implicitly has an 'any' type.
```
ref: https://reviewdog.github.io/errorformat-playground/?efms=%25E%25f+%25%23%28%25l%2C%25c%29%3A+error+TS%25n%3A+%25m&text=src%2Fapp.ts%281%2C29%29%3A+error+TS7006%3A+Parameter+%27x%27+implicitly+has+an+%27any%27+type.%0Asrc%2Fapp.ts%285%2C30%29%3A+error+TS7006%3A+Parameter+%27x%27+implicitly+has+an+%27any%27+type.%0A

This problem does not occur on npx.

--- a/fmts/testdata/tsc.in
+++ b/fmts/testdata/tsc.in
@@ -1,4 +1,7 @@
+yarn run v1.22.10
+$ /Users/odan/source/github.com/odan-sandbox/reviewdog-later-strict-example/node_modules/.bin/tsc --project tsconfig.strict.json --noEmit
 tsc.ts(3,13): error TS7006: Parameter 'y' implicitly has an 'any' type.
 tsc.ts(5,12): error TS2345: Argument of type 'boolean' is not assignable to parameter of type 'number | string'.
   Type 'boolean' is not assignable to type 'string'.
 tsc.ts(11,1): error TS2304: Cannot find name 'error'.
+info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

--- a/fmts/typescript.go
+++ b/fmts/typescript.go
@@ -11,6 +11,7 @@ func init() {
 			`%E%f %#(%l,%c): %m`,       // fallback
 			`%Eerror %m`,
 			`%C%\s%+%m`,
+			`%-G%.%#`,
 		},
 		Description: "TypeScript compiler",
 		URL:         "https://www.typescriptlang.org/",

